bazel-java-deps.bzl

```diff
@@ -173,6 +173,7 @@ def install_java_deps():
             "io.gatling:gatling-http-client:3.3.1",
             "com.fasterxml.jackson.core:jackson-core:2.12.0",
             "com.fasterxml.jackson.core:jackson-databind:2.12.0",
+            "com.softwaremill.diffx:diffx-core_{}:0.4.1".format(scala_major_version),
```

Comment:
can we kill the ai.x.diff dependency now?

Reply:
![kill it with fire](https://media3.giphy.com/media/XvQXEi09zfmcE/giphy.gif?cid=ecf05e47xda0rwb6vnx8bnj67kfqp70exxueemcalp6vronu&rid=giphy.gif)

Reply:
Done 🚮